README.md

```diff
@@ -1,20 +1,20 @@
-# lnd - Lightning Network Daemon
+# Lightning Network Daemon
 
 This repo is preliminary work on a lightning network peer-to-peer node and wallet.
 
 It currently is being designed for testnet-L, a test network where all txids are normalized.  The plan is to eventually use the agreed longer-term solution (segwit?), but we want to not be concerned with updates to segwit, so we're running with this.  Normalized txids fixes malleability but isn't something that will/can be integrated to the existing bitcoin network cleanly (so it's a good way to ensure testing-only before segwit is finalized, which solves the problem too).
 
-This version of Lnd is not yet fully-operational, but a proof of concept on testnet-L will likely exist soon.  The projection is it will be operational before the necessary malleability soft-forks are into bitcoin mainnet (may be significantly before if there are any delays in mainnet malleability fix).
+This version of lightningd is not yet fully-operational, but a proof of concept on testnet-L will likely exist soon.  The projection is it will be operational before the necessary malleability soft-forks are into bitcoin mainnet (may be significantly before if there are any delays in mainnet malleability fix).
 
 Don't try to port it to mainnet or an altcoin and use it today!  No really.  Lightning transactions will be fast, but for now, please wait just a little bit.
 
 ## Installation
 
 * If necessary, install Go according to the installation instructions here: http://golang.org/doc/install. It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
-* Run the following command to obtain and install lnd, lncli, lnshell and all dependencies:
+* Run the following command to obtain and install lightningd, lightningcli, lightningshell and all dependencies:
 
 ```
-go get -u -v github.com/lightningnetwork/lnd/...
+go get -u -v github.com/lightningnetwork/lightningd/...
```

Comment:
Changing this without changing the name of the repo will cause future `go get ...` calls to fail.